### PR TITLE
Add headers for propFind operations

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/evert/davclient.js/issues/14

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -40,18 +40,19 @@ dav.Client.prototype = {
      *
      * @param {string} url Url to do the propfind request on
      * @param {Array} properties List of properties to retrieve.
+     * @param {Object} headers headers
```
Comment (staabm) [inline lib/client.js]: per jsdoc this should be `@param {Object} [headers] headers` as the newly added param is optional

http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

File: lib/client.js [hunk 2]
```diff
@@ -103,6 +104,7 @@ dav.Client.prototype = {
      *
      * @param {string} url Url to do the proppatch request on
      * @param {Array} properties List of properties to store.
+     * @param {Object} headers headers
```
Comment (staabm) [inline lib/client.js]: same here

**Comment by PVince81** [comment]
Done.

**Comment by staabm** [comment]
:+1:

**Comment by LukasReschke** [comment]
:+1:

**Comment by PVince81** [comment]
Thanks guys, merging

**Pull request merged**